pytripgui/controller/main_cont.py

```diff
@@ -94,17 +103,72 @@ def open_voxelplan(self, event):
             logger.debug("   Open '{:s}'".format(vdx_path))
             self.model.vdx = pt.VdxCube(self.model.ctx)
             self.model.vdx.read(vdx_path)
+            vdx = self.model.vdx
 
-        self.app.ctrl.tree.update_tree()
-        self.app.ctrl.plot.update_plot()
+        # add cube to the treeview
+        self.tree.add_vdx(vdx)
 
+        # update the canvas
+        self.plot.update_plot()
         # testing, not sure if this is proper
         # emit signal to update the plot
         # update_plot()
         # from controller.plot_cont import PlotController
         # self.plotupdate.connect(PlotController.update_plot)
         # self.plotupdate.emit()
 
+    def import_dos_dialog(self, event):
+        """
+        Import dose cube.
+        """
+        # Start a file dialog for selecting input files
+        from view.dialogs import MyDialogs
+        dos_path = MyDialogs.openFileNameDialog(self.app)
+        self.import_dos(dos_path)
+
+    def import_dos(self, dos_path):
+        """
+        """
+        logger.debug("Open DosCube {:s}".format(dos_path))
+        dos = pt.DosCube()
+        self.model.dos.append(dos)
+        dos.read(dos_path)
+        # add cube to the treeview
+        self.tree.add_dos(dos)
+
+    def import_let_dialog(self, event):
+        """
+        Import LET cube.
+        """
+        # Start a file dialog for selecting input files
+        from view.dialogs import MyDialogs
```

Comment:
Please fix all imports, they should be like `from pytripgui.view.dialogs import MyDialogs`.
It is needed for proper packaging and distribution of GUI  via pip

Reply:
First this branch should be rebased, as some of the imports are already fixed: https://github.com/pytrip/pytripgui/blob/develop-qt5/pytripgui/controller/plot_cont.py#L3